### DISCUSSION TITLE
新表情建议与讨论专楼

### DISCUSSION THREAD

**Discussion by MeetWq** [MeetWq/meme-generator#192] [想法]

**Comment by saiboxiaogou** [comment]
想要最近很火的那个“剧终”和“未完待续”的meme

**Comment by 127Wzc** [comment]
想要个gif变速的，看他们都是自己另改然后多出一个表情。
是不是可以放到meme作为一个通用的触发。

**Comment by MeetWq** [comment]
> 想要个gif变速的，看他们都是自己另改然后多出一个表情。 是不是可以放到meme作为一个通用的触发。

暂时不打算放到meme里，可以用 [nonebot-plugin-imagetools](https://github.com/noneplugin/nonebot-plugin-imagetools)

**Comment by 127Wzc** [comment]
> > 想要个gif变速的，看他们都是自己另改然后多出一个表情。 是不是可以放到meme作为一个通用的触发。
> 
> 暂时不打算放到meme里，可以用 [nonebot-plugin-imagetools](https://github.com/noneplugin/nonebot-plugin-imagetools)

哦哦原来还有这个库 可以 。👍

**Comment by Li1Fan** [comment]
![扭屁股-1727147002656](https://github.com/user-attachments/assets/c0d013da-9026-499e-8319-8595d8018e54)
![呼啦圈-1727146994043](https://github.com/user-attachments/assets/89c7938a-d4ba-44e4-a124-e9c994947897)
![大摇大摆-1727146981632](https://github.com/user-attachments/assets/1a164d87-84be-4123-8c1f-639dccb5c41d)
![可乐-1727146928399](https://github.com/user-attachments/assets/05075266-c8f9-4e60-bfef-52c847f6960f)

建议加一下这四个表情，跟gif有关的代码，不太会写

**Comment by Anthoooooooony** [comment]
建议！
![image](https://github.com/user-attachments/assets/ce234527-6613-4214-b9ef-3dd39daa4912)

**Comment by CampioneBase** [comment]
下一步打算制作一个《舌战群儒——旧病复发.gif》
![03f3afbacdffecbe6e6de6b8415d24a7](https://github.com/user-attachments/assets/b244c305-a2cf-438f-8e62-8a2cda157b8d)

**Comment by monbed** [comment]
建议：咖波，你不懂啦。
![你不懂啦](https://github.com/user-attachments/assets/6acf020c-fe36-4646-a60b-add4a1a741bf)
![f7190a4bb67d8e91d49588a49ac2a3e9](https://github.com/user-attachments/assets/a31bfd14-c887-4133-b143-c7eabffacfad)
![image](https://github.com/user-attachments/assets/4a08f445-a03f-4794-a7ca-84258c952490)

**Comment by MobiusT** [comment]
https://github.com/user-attachments/assets/c2a1a0bf-4c35-47d2-9bb8-8852dfbeb281

**Comment by MobiusT** [comment]
![Image_1751298612398.png](https://github.com/user-attachments/assets/8efe59bc-2a08-4e99-9ce7-bfb093ad440c)

![Image_1751298609036.png](https://github.com/user-attachments/assets/702c7042-71ed-4ab2-9844-f2dd4f997020)

![Image_1751298606656.png](https://github.com/user-attachments/assets/aa41c996-e3da-47f4-9075-62c5db339dbd)

![Image_1751298603460.png](https://github.com/user-attachments/assets/4b4d1c6c-aff3-4248-8f38-c29359bbc5f2)

![Image_1751298599966.png](https://github.com/user-attachments/assets/5a7258b1-32d6-43d2-bb43-0a7b7fd1d031)

![Image_1751298596771.png](https://github.com/user-attachments/assets/54759348-96bb-4fd6-8a74-4fbb59038cf0)

![Image_1751298593193.png](https://github.com/user-attachments/assets/3b1a769c-b124-48b7-b6b3-3654737bb4a0)

**Comment by MobiusT** [comment]
![08B6AE047C2AA01C40A2F7BF9CBB6B49](https://github.com/user-attachments/assets/1f697f08-9c90-4541-acf2-6f802c83da34)
